.github/workflows/tests.yml

```diff
@@ -22,9 +20,9 @@ jobs:
         - python-version: "3.12"
           env:
             TOXENV: py
-        - python-version: pypy3.9
+        - python-version: "3.13"
           env:
-            TOXENV: pypy
+            TOXENV: py
         - python-version: pypy3.10
           env:
             TOXENV: pypy
```

Comment:
Before we had the lowest and highest supported PyPy versions. Now there is only PyPy 3.10. Are we dropping support for PyPy 3.9?